models/user.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require('mongoose');
+
+const User = new mongoose.Schema({
+    username: {
+        type: String,
+        required: true,
+        maxlength: 255
+    },
+    password: String,
```

Comment:
Does the password not need to be required like the other fields? Honestly asking because I don't know